ddp-workspace/projects/toolkit/src/lib/models/privacyModalData.ts

```diff
@@ -0,0 +1,3 @@
+export class PrivacyModalData {
+  constructor(public usePrionTemplate: boolean) { }
```

Comment:
Reference to Prion in toolkit. Could this be generalized to so it is not prion-specific? If not or too hard or too much work, then probably we should move all the privacy modal stuff to either ddp-prion or toolkit-prion.

Reply:
The challenge with this is that since different parts of the privacy policy have different styles and since the privacy policy contains buttons, making all studies use Prion's template is going to be rigid to the point of making the whole thing useless.  If I put Prion's template in toolkit-prion or ddp-prion, toolkit would have to import from toolkit-prion or ddp-prion, making a circular dependency.  So it seems like for the study-specific privacy policy to be in the study-specific module, anything that uses it would also need to be in the module.  The issue here is that this ends up being a lot of stuff: prion-privacy-policy is used in toolkit-privacy-policy-modal, which is used in PrivacyPolicyButtonComponent, which is used in the cookie banner AND the cookie preferences modal.  In other words, all the GDPR features would need to be duplicated for any study that wants to use them.  The only other option I can think of is having an entire module with just Prion's privacy policy (and potentially additional modules for each study's privacy policy) to prevent the circular dependency.  Probably the real answer to this is to make the privacy policy something that can be configured in the study builder, but I don't have enough time for that at this point.  So I'm not sure what to do about this.